python/mead/export.py

```diff
@@ -31,7 +32,7 @@ def main():
     task = mead.Task.get_task_specific(task_name, args.logging, args.settings)
     task.read_config(config_params, args.datasets)
     exporter = create_exporter(task, args.exporter_type)
-    exporter.run(args.model, args.output_dir, args.model_version)
+    exporter.run(args.model, args.output_dir, args.model_version, preproc=args.preproc)
```

Comment:
It doesnt make sense that the exporter should have an additional argument for preproc -- that should be *the function* of a preproc exporter